tox.ini

```diff
@@ -49,7 +50,7 @@ ignore =
 require-code = True
 
 [doc8]
-ignore-path=.tox,src/sagemaker.egg-info
```

Comment:
doc8, is complaining about some file in `src/sagemaker.egg-info` may need to add this part back

Reply:
Done, thanks!